diff --git a/bin/prokka b/bin/prokka
index fb192d0..d731af3 100755
--- a/bin/prokka
+++ b/bin/prokka
@@ -700,7 +700,7 @@ if ($tools{'minced'}->{HAVE}) {
 
 msg("Predicting coding sequences");
 my $totalbp = sum( map { $seq{$_}{DNA}->length } @seq);
-my $prodigal_mode = ($totalbp >= 100000 && !$metagenome) ? 'single' : 'meta';
+my $prodigal_mode = !$metagenome ? 'single' : 'meta';
 msg("Contigs total $totalbp bp, so using $prodigal_mode mode");
 my $num_cds=0;
 my $cmd = "prodigal -i \Q$outdir/$prefix.fna\E -c -m -g $gcode -p $prodigal_mode -f sco -q";
